src/main/java/com/ldtteam/structurize/items/ItemCaliper.java

```diff
@@ -21,10 +21,8 @@
     private static final RangedAttribute ATTRIBUTE_CALIPER_USE = new RangedAttribute((IAttribute) null, "player.caliperUse", 0.0, 0.0, 1.0);
 
     private static final double HALF                        = 0.5;
-    private static final String ITEM_CALIPER_MESSAGE_LINE   = "item.caliper.message.line";
-    private static final String ITEM_CALIPER_MESSAGE_SQUARE = "item.caliper.message.square";
-    private static final String ITEM_CALIPER_MESSAGE_CUBE   = "item.caliper.message.cube";
     private static final String ITEM_CALIPER_MESSAGE_SAME   = "item.caliper.message.same";
+    private static final String[] shapes = {"line", "square", "cube"};
```

Comment:
Make this an enum.

Reply:
This will be cleaned later, it needs a better system for translating

Reply:
And not sure why this should be an enum though

Reply:
Since it is a Collection of shapes. Never ever use inline constants.

Reply:
I agree with Orion. Enum makes this nicer

Reply:
I find using enum here a bit overkill when they are just strings used to create printable string... Will make the caliper stuffs translatable soon (in this PR)

Reply:
What do you mean? Enums are pretty simple, and way better than using shapes[1]. Shapes.SQUARE is much more clear.

Reply:
I don't use shapes[1] anywhere, I just do `stringBuilder.append(shapes[flag]);`, whatever I moved the discussion to discord since I will rework this to accept translations thus this will disappear